docker_test.go

```diff
@@ -443,18 +523,117 @@ func TestWebhook(t *testing.T) {
 	require.Equal(t, gjson.GetBytes(body, "anonymousId").Str, "anon-id-new")
 	require.Equal(t, gjson.GetBytes(body, "userId").Str, "identified user id")
 	require.Equal(t, gjson.GetBytes(body, "rudderId").Str, "daf823fb-e8d3-413a-8313-d34cd756f968")
-	require.Equal(t, gjson.GetBytes(body, "type").Str, "track")
-
-
-	// TODO: Verify in Live Evets API
+	require.Equal(t, gjson.GetBytes(body, "type").Str, "identify")
+	
 }
 // Verify Event in POSTGRES
 func TestPostgres(t *testing.T) {
 	var myEvent Event
 	require.Eventually(t, func() bool {
-		eventSql:= "select anonymous_id, user_id from example.tracks limit 1"
+		eventSql:= "select anonymous_id, user_id from example.identifies limit 1"
 		db.QueryRow(eventSql).Scan(&myEvent.anonymous_id, &myEvent.user_id)
 		return myEvent.anonymous_id == "anon-id-new"
 	}, time.Minute, 10*time.Millisecond)
 	require.Equal(t, "identified user id", myEvent.user_id)
+	
+	}
+// Verify Event in Redis	
+func TestRedis(t *testing.T) {
+	fmt.Println(address)
+	conn, err := redigo.Dial("tcp", address)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer conn.Close()
+	require.Eventually(t, func() bool {
+		// Similarly, get the trait1 and convert it to a string.
+		event, _ := redigo.String(conn.Do("HGET", "user:identified user id", "trait1"))
+		return event == "new-val"
```

Comment:
No that is important here, but to make linters happy:
```suggestion
		event, err := redigo.String(conn.Do("HGET", "user:identified user id", "trait1"))
		return err == nil && event == "new-val"
```